run/helloworld/e2e_test.go

```diff
@@ -0,0 +1,49 @@
+package main_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/cloudrunci"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
```

Comment:
We might want to consider moving the `cloudrunci` tests to a separate directory so the samples don't have to depend on golang-samples. Can be a separate PR.

Reply:
👍  will file a follow-up issue to discuss.